ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_matrix_and_vector_products.py

```diff
@@ -75,3 +75,82 @@ def test_numpy_outer(
         b=np.array(y, dtype=input_dtype2),
         out=None,
     )
+    
+
+import numpy as np
```

Comment:
Same for all of these imports. All imports go at the top of the file and there should be no duplication.

Reply:
@simonetgordon  I have corrected that.